3424 - Fix the overlay with modal and busy indicator (#3600)
infor-design/enterprise: docs/CHANGELOG.md

- `[Bar Chart]` Fixed an issue where chart was not resizing on homepage widget resize. ([#2669](https://github.com/infor-design/enterprise/issues/2669))
- `[Blockgrid]` Fixed an issue where there was no index if the data is empty, and removed deprecated internal calls. ([#748](https://github.com/infor-design/enterprise-ng/issues/748))
- `[Busy Indicator]` Fixed an issue where it throws an error when a display delay, the busy-indicator parent removed and added via ngIf before the busyindicator shown. ([#703](https://github.com/infor-design/enterprise-ng/issues/703))

> - `[Busy Indicator]` Fixed an issue where the overlay was closing after close Modal. ([#3424](https://github.com/infor-design/enterprise/issues/3424))

Reviewer: ```suggestion
- `[Busy Indicator]` Fixed an issue where the overlay would close when closing the Modal. ([#3424](https://github.com/infor-design/enterprise/issues/3424))
```
